occurrences/notification_services.py

```diff
@@ -17,16 +18,39 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_enrolment_summary_report_data(days=1):
+    enrolments = (
+        occurrences_models.Enrolment.objects.pending_and_auto_accepted_enrolments(
+            days=days
+        ).select_related("occurrence", "occurrence__p_event")
+    )
+    queued_enrolments = (
+        occurrences_models.EventQueueEnrolment.objects.enrolled_in_last_days(days=days)
+    )
+    return {"enrolments": enrolments, "queued_enrolments": queued_enrolments}
+
+
+def send_enrolment_summary_report_to_providers_from_days(days=1):
+    send_enrolment_summary_report_to_providers(
+        **_get_enrolment_summary_report_data(days=days)
+    )
```

Comment:
Supporting Zen of Python's
- "Explicit is better than implicit"
- "Flat is better than nested"

I think not putting the enrolments & queued enrolments into a dictionary put passing them explicitly as the named parameters they are going to would be clearer here IMO:

```suggestion
def _get_enrolments_for_summary_report(days=1):
    return (
        occurrences_models.Enrolment.objects.pending_and_auto_accepted_enrolments(
            days=days
        ).select_related("occurrence", "occurrence__p_event")
    )

        
def _get_queued_enrolments_for_summary_report(days=1):
    return (
        occurrences_models.EventQueueEnrolment.objects.enrolled_in_last_days(days=days)
    )


def send_enrolment_summary_report_to_providers_from_days(days=1):
    send_enrolment_summary_report_to_providers(
        enrolments=_get_enrolments_for_summary_report(days),
        queued_enrolments=_get_queued_enrolments_for_summary_report(days),
    )
```